src/RenderWebGL.js

```diff
@@ -496,17 +540,15 @@ class RenderWebGL {
             gl.disable(gl.STENCIL_TEST);
         }
 
-        const pixels = new Buffer(bounds.width * bounds.height * 4);
+        const pixels = new Uint8Array(bounds.width * bounds.height * 4);
```

Comment:
Do you know what babel will convert this to? `Uint8Array` is currently only supported by Chrome and Firefox. Perhaps something we can polyfill?